docs/paper.md

```diff
@@ -37,30 +37,30 @@ bibliography: paper.bib
 # Summary
 
 Simulating multi-physics phenomena for real-world applications states various challenges in scientific computing.
-The behavior of individual physical domains involved is often described through distinct partial-differential equations, which need to be solved in a certain subdomain.
+The behavior of individual physical domains involved are often described through distinct partial-differential equations, which need to be solved in a certain subdomain.
```

Comment:
The verb is supposed to refer to "The behavior", which is singular. So I would expect `is` to be the correct grammar here, no?

Reply:
To me, the subject is the behavior of the (multiple) domains, which I see as plural. If this was the behavior of a single domain, I would agree with you.

Reply:
Overall, there are multiple physical domains involved. In the sentence above, we refer to each individual domain, i.e., one behavior per domain. Honestly speaking, I am not a native English speaker, so I am not entirely sure what's correct in this scenario. If you understand it in this way as plural, I am good to go with 'are'.

Reply:
What about
> Each individual physical domain has behavior that is often described through a distinct set of partial-differential equations that needs to be solved in that domain.

Reply:
Perfect! I propagated the rephrasing also to the subsequent sentence.